src/main/java/com/sapozhnykov/domain/Order.java

```diff
@@ -0,0 +1,52 @@
+package com.sapozhnykov.domain;
+
+import java.lang.reflect.Array;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    private static long id = 1;
+    private long idClient;
+    private BigDecimal finalCost;
```

Comment:
may be it's sum or amount?

Reply:
At the last lesson, Artem said that our Order would not have a COST . I removed it from the project.